db/migrate/20221116182526_create_apple_credentials.rb

```diff
@@ -0,0 +1,20 @@
+class CreateAppleCredentials < ActiveRecord::Migration
+  def change
+
```

Comment:
Some extra/blank lines in this file?  Generally, I'd only leave a space where you're trying to call out some logical difference between pieces of code.  (Like between the `create_table` block here, and the 2 `add_foreign_key`s below it).

Reply:
We don't generally want to edit migrations directly, if I'm remembering right...? Or am I thinking of something else?

Reply:
You're thinking of `schema.rb` - which gets automatically changed when you run `db:migrate`.